### PR TITLE
Release v3.9.0

### PR DESCRIPTION
**Pull request by Dreamsorcerer**

### CODE REVIEW

**Comment by webknjaz** [comment]
Minor note: I usually try to go through the changelog to make sure it's cohesive, uses the same writing style and so on. After running towncrier, that is. The PR RTD build is usually a good place for previewing if it looks good.

File: CHANGES.rst [hunk 1]
```diff
@@ -271,6 +161,58 @@ Bugfixes
 
   `#7616 <https://github.com/aio-libs/aiohttp/issues/7616>`_
 
+- Fixed a rare `RuntimeError: await wasn't used with future` exception -- by :user:`stalkerg`
```
Comment (Dreamsorcerer) [inline CHANGES.rst]:
```suggestion
- Fixed a rare ``RuntimeError: await wasn't used with future`` exception -- by :user:`stalkerg`
```

**Comment by Dreamsorcerer** [comment]
I'm going to let the wheels build overnight, will approve the deploy when I'm up tomorrow.

**Comment by webknjaz** [comment]
I wasn't sleeping when the notification came so I pushed the button.

**Pull request merged**